src/Network/Network.Test/ScenarioTests/ExpressRoutePortTests.cs

```diff
@@ -32,5 +32,13 @@ public void TestExpressRoutePortCRUDMinimalParameters()
         {
             TestRunner.RunTestScript(string.Format("Test-ExpressRoutePortCRUD"));
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.Owner, NrpTeamAlias.pgtm)]
+        public void TestExpressRoutePortIdentityCRUD()
```

Comment:
You're missing test recording for this test

Reply:
Added recordings